source/mainnet/net/guides/developer-page.rst

```diff
@@ -53,3 +53,4 @@ Community resources
     sdks-apis
     ../resources/dashboards
     ../references/developer-tools
+    ../references/grpc
```

Comment:
We need to update the README to describe how the schemas were generated so that the page can be maintained.